### ISSUE TITLE
Config parsing trips on spaces?

### ISSUE THREAD

**Issue by lukasheinrich** [scikit-hep/pyhf#1389]
# Description

@elskorda has a workspace where the POI name stored in the JSON ends up being `'SigXsecOverSM '` (notice the space). This seems to be a bug in our parsing (related to eventually supporting multi pos with space-separated names)

```
<!DOCTYPE Combination  SYSTEM 'HistFactorySchema.dtd'>

<Combination OutputFilePrefix="output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/output">

<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax200_BMin150_incFat1_Y2015_T2_J2_L0_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax350_BMin200_incFat1_Y2015_T2_J2_L0_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax500_BMin350_incFat1_Y2015_T2_J2_L0_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax750_BMin500_incFat1_Y2015_T20_J0_L0_incJet1_Fat1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMin750_incFat1_Y2015_T20_J0_L0_incJet1_Fat1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax200_BMin150_incFat1_Fat0_T3_Y2015_incTag1_J2_L0_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax350_BMin200_incFat1_Fat0_T3_Y2015_incTag1_J2_L0_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMax500_BMin350_incFat1_Fat0_T3_Y2015_incTag1_J2_L0_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DSR_BMin500_incFat1_Fat1_T21_Y2015_incTag1_J0_L0_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax200_BMin150_incFat1_Y2015_T2_J2_L1_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax350_BMin200_incFat1_Y2015_T2_J2_L1_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax500_BMin350_incFat1_Y2015_T2_J2_L1_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMin500_incFat1_Y2015_T20_J0_L1_incJet1_Fat1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax200_BMin150_incFat1_Fat0_T3_Y2015_incTag1_J2_L1_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax350_BMin200_incFat1_Fat0_T3_Y2015_incTag1_J2_L1_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMax500_BMin350_incFat1_Fat0_T3_Y2015_incTag1_J2_L1_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distCharge_DCR1_BMin500_incFat1_Fat1_T21_Y2015_incTag1_J0_L1_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax200_BMin150_incFat1_Y2015_T2_J2_L2_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax350_BMin200_incFat1_Y2015_T2_J2_L2_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax500_BMin350_incFat1_Y2015_T2_J2_L2_incJet1_Fat0.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMin500_incFat1_Y2015_T20_J0_L2_incJet1_Fat1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax200_BMin150_incFat1_Fat0_T3_Y2015_incTag1_J2_L2_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax350_BMin200_incFat1_Fat0_T3_Y2015_incTag1_J2_L2_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMax500_BMin350_incFat1_Fat0_T3_Y2015_incTag1_J2_L2_incJet1.xml</Input>
<Input>output/MonoHbb-00-13-00-Paper-ExtendedGrid.monoH_1300_unblinded/xml/1000/Region_distmBB_DCR2_BMin500_incFat1_Fat1_T21_Y2015_incTag1_J0_L2_incJet1.xml</Input>

<Measurement Name="VH" Lumi="1" LumiRelErr="0.0001" ExportOnly="True">
<POI>SigXsecOverSM </POI>
<ParamSetting Const="True">Lumi </ParamSetting>
</Measurement>
</Combination>
```

**Comment by matthewfeickert** [comment]
> This seems to be a bug in our parsing (related to eventually supporting multi pos with space-separated names)

What's the actual error? Does it just fail or does it do just create nonsense?

**Comment by lukasheinrich** [comment]
NB: the same is true for parsing parameter configs

**Comment by matthewfeickert** [comment]
@elskorda This has been resolved now in PR #1436 (thanks @lukasheinrich!). This will be out in an officially supported release in `v0.6.2`, but if you want to use something unsupported until then you can either [install `pyhf` from GitHub](https://pyhf.readthedocs.io/en/v0.6.1/installation.html#install-latest-development-version-from-github) or you can [install from TestPyPI](https://pyhf.readthedocs.io/en/v0.6.1/development.html#testpypi).

I'll stress though that we only support official releases and so development releases might break your workflow. ;)
